conda_smithy/templates/run_docker_build.tmpl

```diff
@@ -26,9 +26,12 @@ show_channel_urls: true
 CONDARC
 )
 
+HOST_USER_ID=$( hash docker-machine 2>/dev/null && docker-machine ssh $(docker-machine active) 'id -u $(whoami)' || id -u $(whoami) )
```

Comment:
Though that is kind of tricky as that avoids using `tini` to reap zombie processes and doesn't source `conda` or `devtoolset`. Maybe we should just check if `$HOST_USER_ID` is `0`. If it is, we skip creating a user and changing everything over.